dev/io.openliberty.checkpoint_fat_faces/bnd.bnd

```diff
@@ -0,0 +1,69 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src,\
+	test-applications/CDIIntegrationTest.war/src, \
+	test-applications/CDIManagedProperty.war/src, \
+	test-applications/ConvertDateTime.war/src, \
+	test-applications/ConverterValidatorBehaviorInjectionTarget.war/src, \
+	test-applications/ELImplicitObjectsViaCDI.war/src, \
+	test-applications/ELImplicitObjectsViaCDI.war/src, \
+	test-applications/ELImplicitObjectsViaCDIErrorApp.war/src, \
+	test-applications/UIRepeatConditionCheck.war/src, \
+	test-applications/PostRenderViewEvent.war/src
+
+fat.project: true
+
+tested.features: \
+    beanvalidation-2.0,\
+    beanvalidation-3.0,\
+    cdi-2.0,\
+    cdi-3.0,\
+    cdi-4.0,\
+    cdi-4.1,\
+    checkpoint,\
+    expressionlanguage-4.0,\
+    expressionlanguage-5.0,\
+    expressionlanguage-6.0,\
+    faces-3.0,\
+    faces-4.0,\
+    faces-4.1,\
+    jsf-2.3,\
+    pages-3.0,\
+    pages-3.1,\
+    servlet-4.0,\
+    servlet-5.0,\
+    servlet-6.0,\
+    servlet-6.1,\
+    validation-3.1
```

Comment:
you need to add `checkpoint` to the list of `tested.features` so the test will run in the checkpoint run of the build.

Reply:
nevermind, `checkpoint` is there already.